bridge/cypress/support/pageobjects/SequencesPage.ts

```diff
@@ -75,4 +75,9 @@ export class SequencesPage {
       .should(exists ? 'exist' : 'not.exist');
     return this;
   }
+
+  public assertServiceName(name: string, tag?: string) {
```

Comment:
Linting fails because the return type is missing. `this` should be returned here because of chaining.

Reply:
Ah I see, thanks I've added this here: https://github.com/keptn/keptn/pull/7283/commits/9fe6175d55aeb44f85ad4bfc175b55b5f32ef141